image-builder/image_builder.sh

```diff
@@ -39,8 +40,19 @@ Options:
 Extra environment variables:
 	AGENT_BIN:  use it to change the expected agent binary name
 	AGENT_INIT: use kata agent as init process
+	DAX: If 'yes' will build the image with DAX support. The resulting image
+	     *can't* be mounted because the first 2M are reserved for the device
```

Comment:
Eeek - I think this is going to break https://github.com/kata-containers/runtime/blob/master/data/kata-collect-data.sh.in then unless we `losetup -o <offset> ...` ?

The problem is, how is `kata-collect-data.sh` going to know to apply the offset? We could add in an arch check but I'm starting to think we need a metadata file for each image (basically `osbuilder.yaml` + extra fields that would live *outside* the VM in the assets directory alongside the images.

Reply:
my bad, after reading `losetup` documentation, now I see that it's possible to mount the image using `-o offset`

Reply:
Yep - the problem is knowing when to use that option though.